app/presenters/migration/index.rb

```diff
@@ -5,16 +5,55 @@ class Index < ApplicationPresenter
     def content
```

Comment:
This page is missing its title:
```html
<title>translation missing: en.contribute.campaigns.migration.title - Europeana Contribute</title>
```

And second crumb in its breadcrumb:
![screenshot_2018-02-26_12-16-12](https://user-images.githubusercontent.com/218337/36670072-ea26ce1e-1aee-11e8-8513-9eb23aeeb3bc.png)